api/src/test/java/uk/gov/dhsc/htbhf/claimant/communications/PaymentCycleNotificationHandlerTest.java

```diff
@@ -14,34 +16,55 @@
 import uk.gov.dhsc.htbhf.claimant.message.payload.EmailType;
 import uk.gov.dhsc.htbhf.claimant.message.payload.MessagePayload;
 
+import java.time.LocalDate;
+
+import static java.util.Arrays.asList;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.BDDMockito.eq;
 import static org.mockito.BDDMockito.verify;
 import static uk.gov.dhsc.htbhf.claimant.message.EmailPayloadAssertions.assertEmailPayloadCorrectForClaimantWithAllVouchers;
+import static uk.gov.dhsc.htbhf.claimant.message.EmailPayloadAssertions.assertThatEmailPayloadCorrectForBackdatedPayment;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.ClaimTestDataFactory.aClaimWithExpectedDeliveryDate;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.PaymentCycleTestDataFactory.aPaymentCycleWithCycleEntitlementAndClaim;
 import static uk.gov.dhsc.htbhf.claimant.testsupport.PaymentCycleTestDataFactory.aValidPaymentCycle;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.PaymentCycleVoucherEntitlementTestDataFactory.aPaymentCycleVoucherEntitlementWithBackdatedVouchersForYoungestChild;
 
 @ExtendWith(MockitoExtension.class)
-class PaymentCycleNotificationEmailHandlerTest {
+class PaymentCycleNotificationHandlerTest {
 
     @Mock
     private MessageQueueClient messageQueueClient;
     @Mock
     private UpcomingBirthdayEmailHandler upcomingBirthdayEmailHandler;
 
     @InjectMocks
-    PaymentCycleNotificationEmailHandler paymentCycleNotificationEmailHandler;
+    PaymentCycleNotificationHandler paymentCycleNotificationHandler;
 
     @Test
     public void shouldSendEmailAndInvokeUpcomingBirthdayEmailHandler() {
         PaymentCycle paymentCycle = aValidPaymentCycle();
         Claim claim = paymentCycle.getClaim();
 
-        paymentCycleNotificationEmailHandler.sendNotificationEmails(paymentCycle);
+        paymentCycleNotificationHandler.sendNotificationEmails(paymentCycle);
 
         verifyPaymentEmailNotificationSent(paymentCycle, claim);
         verify(upcomingBirthdayEmailHandler).handleUpcomingBirthdayEmails(paymentCycle);
     }
 
+    @Disabled("HTBHF-2028")
+    @Test
+    public void shouldSendNewChildFromPregnancyEmail() {
+        Claim claim = aClaimWithExpectedDeliveryDate(LocalDate.now().minusWeeks(8));
+        PaymentCycleVoucherEntitlement voucherEntitlement =
+                aPaymentCycleVoucherEntitlementWithBackdatedVouchersForYoungestChild(LocalDate.now(), asList(LocalDate.now().minusWeeks(6)));
+        PaymentCycle paymentCycle = aPaymentCycleWithCycleEntitlementAndClaim(voucherEntitlement, claim);
```

Comment:
There is a bug in `aPaymentCycleWithCycleEntitlementAndClaim`. It needs updating to:

```
return aValidPaymentCycleBuilder()
                .voucherEntitlement(paymentCycleVoucherEntitlement)
                .totalVouchers(paymentCycleVoucherEntitlement.getTotalVoucherEntitlement()) // <- this is new
                .totalEntitlementAmountInPence(paymentCycleVoucherEntitlement.getTotalVoucherValueInPence()) // <- this is new
                .claim(claim)
                .childrenDob(nullSafeGetChildrenDob(claim))
                .build();
```

Reply:
Done - I made the same change in all the other methods in that factory too.